src/app.tsx

```diff
@@ -197,6 +224,124 @@ const SearchReplaceForBlockEditor = () => {
     }
   }
 
+  /**
```

Comment:
@anandraj346 This change with the **Table** block has a lot going on, and should be a `feature` because the previous implementation: `replaceBlockAttribute` did not accurately capture it's edge-cases. Pls let's back out this **Table** block change in this PR and create a new PR for it specifically to be pointed to `release-1.3.0` branch.

Reply:
Ok sure